go/bind/keybase.go

```diff
@@ -141,14 +142,15 @@ func Init(homeDir string, logFile string, runModeStr string, accessGroupOverride
 	}
 
 	go func() {
-		kbfsParams := libkbfs.DefaultInitParams(kbCtx)
+		kbfsCtx := env.NewContextFromGlobalContext(kbCtx)
```

Comment:
Can you pass in `libkb.G` here, since you're using it just above? That should preserve the current behavior. This PR changes the behavior by instantiating a second `libkb.GlobalContext`.

Reply:
As far as I can tell, `kbCtx` _is_ `libkb.G`. Passing in `libkb.G` here is the same thing, isn't it?

Reply:
Gah, you're right, I misread it.  Sounds good.